program_intent_engagement/apps/api/v1/urls.py

```diff
@@ -1,12 +1,19 @@
 """ API v1 URLs. """
 from django.urls import re_path
 
-from program_intent_engagement.apps.api.v1.views import ProgramIntentsView
```

Comment:
I'm curious why you changed this to a relative import. Absolute imports are generally favored. Was it the linter?

Reply:
I honestly didn't think too much of it, during merge one of the imports was relative and another absolute and I had changed both to relative since I thought it was preferred. This is good to know and i've updated this to be absolute 👍